lib/registries/addon/components/registries-navbar/template.hbs

```diff
@@ -1,105 +1,192 @@
-<Navbar @dark={{true}} as |nav|>
+<Navbar
+    data-analytics-scope='Navbar'
+    @dark={{true}}
+    ...attributes
+    as |nav|
+>
     <nav.container>
-        {{!-- Left Side (Band, Service Dropdown) --}}
+        {{! Left Side (Band, Service Dropdown) }}
         <nav.section data-test-left>
 
-            {{#global-link-to data-test-brand 'home' (html-attributes aria-label=(t 'navbar.go_home'))}}
-                <div local-class="Logo"></div>
-            {{/global-link-to}}
-
-            {{#bs-dropdown as |dd|}}
-                {{#dd.toggle data-test-service classNames=(local-class 'Service Dropdown')}}
-                    <span local-class="HideOnMobile">{{t 'general.OSF'}}</span>
+            <Link
+                data-analytics-name='Brand'
+                data-test-brand
```

Comment:
We've been putting the data-test as the first item in the component partially because it had to be for curly-brace components but mostly because it makes for easy identification of the component.

Reply:
I'll change that to be consistent, but it does feel like `data-test-` and `data-analytics-name` are now mildly redundant...

Reply:
Yeah, I don't see any reason not to get rid of data-test on elements that have data-analytics if you change the tests and inform QA about the change so they can also update.